Enigmatry.Entry.BlobStorage/Azure/AzureBlobSharedUri.cs

```diff
@@ -34,9 +34,8 @@ public static bool TryParse(Uri uri, out AzureBlobSharedUri sharedUri)
 
         private static AzureBlobSharedUri Parse(Uri uri)
         {
-            var decodedUri = new Uri(HttpUtility.UrlDecode(uri.AbsoluteUri));
```

Comment:
Removed double decoding, no need to decode full URL. Only query params should be decoded which we already do.